test/known_issues/test_mixing_zero_energy_cuts.py

```diff
@@ -17,8 +17,8 @@ def test_mix_zero_energy_cut_raises(self, snr):
 
         mixed = zero_cut.mix(rand_cut, snr=snr)
 
-        with pytest.raises(NonPositiveEnergyError):
-            mixed.load_audio()
+        mix_cut_samples = mixed.load_audio()
+        assert mix_cut_samples.shape[1] == sr
```

Comment:
can you additionally assert that `np.testing.assert_equal(rand_cut.load_audio(), mix_cut_sampler)`?